docs/data/sdks/java/index.md

```diff
@@ -46,9 +46,33 @@ Amplitude client = Amplitude.getInstance("YOUR_INSTANCE_NAME");
 client.init("YOUR_API_KEY");
 ```
 
-### Configure batching behavior
-
-To support high performance environments, the SDK sends events in batches. Every event logged by `logEvent` method is queued in memory. Events are flushed in batch in background. You can customize batch behavior with `setEventUpdloadThreshfold` and `setEventUploadPeriodMillis`.
+### Configuration
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div>  | Description | Default Value |
+    | --- | --- | --- |
+    | `setServerUrl` | The server url events are uploaded to. | `https://api2.amplitude.com/2/httpapi` |
+    | `useBatchMode` |  Whether to use [batch api](../../../analytics/apis/batch-event-upload-api/#batch-event-upload). By default, the SDK will use the default `serverUrl` | `false` |
+    | `setLogMode` |  The level at which to filter out debug messages. | `LogMode.ERROR` |
+    | `setEventUploadThreshold` |  SDK will attempt to upload once unsent event count exceeds the event upload threshold or reach eventUploadPeriodSeconds interval. | `10` |
+    | `setEventUploadPeriodMillis` | The amount of time SDK will attempt to upload the unsent events to the server or reach eventUploadThreshold threshold. | `10 seconds` |
+    | `setCallbacks` | Event callback which are triggered after event sent. | `null`|
+    | `setProxy` | Custom proxy for https requests. | `Proxy.NO_PROXY`|
+    | `setFlushTimeout` | Events flushing thread timeout in milliseconds. | `0` |
+    | `setOptions` | A dictionary of key-value pairs that represent additional instructions for server save operation.| Please check [available options](data/sdks/java/#options). |
+
+### Options
+
+???config "Availabe Options"
+    | <div class="big-column">Name</div>  | Description | Default Value |
+    | --- | --- | --- |
+    | `Options.setMinIdLength` | Set the minimum length for user id or device id. | 5 |
+    | `Options.setHeaders` | Set the custom headers. | `{"Content-Type", "application/json","Accept", "application/json"}` |
+    | `Options.addHeaders` | Add more custom headers. | `{"Content-Type", "application/json", "Accept", "application/json"}` |
+
+#### Configure batching behavior
+
+To support high performance environments, the SDK sends events in batches. Every event logged by `logEvent` method is queued in memory. Events are flushed in batch in background. You can customize batch behavior with `setEventUpdloadThreshfold` and `setEventUploadPeriodMillis`. By default, the serverUrl will be `https://api2.amplitude.com/2/httpapi`. For customers who want to send large batches of data at a time. please check the `useBatchMode`. Both the regular mode and the batch mode use the same events upload threshold and flush time intervals.
```

Comment:
Added more words for avoiding confusion between `batch` and `batch mode`. But it's has some overlap with the following doc. Let me know what do you think.
<img width="783" alt="Screen Shot 2023-03-15 at 11 53 33 AM" src="https://user-images.githubusercontent.com/13028329/225414782-905059f8-8ffc-4147-a3c8-85082c08f4c4.png">

Reply:
How about explicitly saying the main difference between two modes is which `serverUrl` they use? 

> For customers who want to send large batches of data at a time, set`useBatchMode` to `true` to set `setServerUrl` to batch event upload api `https://api2.amplitude.com/batch`.

Reply:
Thanks @Mercy811 I think mentioning the separate endpoint is a good idea